client/my-sites/stats/stats-purchase/tier-upgrade-slider/index.tsx

```diff
@@ -84,6 +86,10 @@ function TierUpgradeSlider( {
 	// Only visible if the slider is at the max value and we have a string/node to display.
 	const infoReferenceElement = useRef( null );
 	const showPopup = currentPlanIndex === sliderMax && popupInfoString !== undefined;
+
+	const firstTierInfoRef = useRef( null );
+	const showFirstTierInfo = currentPlanIndex === 0 && firstTierInfo !== undefined;
```

Comment:
Can we please not open the tooltip automatically? Maybe we should remove the automatic popup for the last tier too. Both of them are very noisy especially on mobile. What do you think?

Reply:
I noticed the automatic popup was a little weird when we tackled the last tier, but it also made users pay attention.

> Both of them are very noisy especially on mobile.

Indeed, I can imagine.

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/89764/commits/ce8294ec6150b77c24443d93a6579ef46ed8be1a and https://github.com/Automattic/wp-calypso/pull/89764/commits/ea0a117f243245ca864c5b01aead81e2effc19f4.